sdk/search/search-documents/recordings/node/searchindexerclient_skillsets/recording_throws_error_for_invalid_skillset_object.js

```diff
@@ -1,6 +1,6 @@
 let nock = require('nock');
 
-module.exports.hash = "2cbb4a8bd8e0507fca06bbaff714dac2";
+module.exports.hash = "935c88a3b3a2c1559ec672f424dfd555";
```

Comment:
I'm sorry for these unnecessary recordings 😓

Reply:
Briefly saw some of the recordings.. From the changes, it looks like we don't need these updates to the recordings.

Reply:
@sarangan12, maybe you can revert the changes to the recordings.
Generate recordings only for the tests that are affected.

Reply:
originally I thought the deletions might impact recordings. They do not. Since it has been a month old anyway, I just updated all of them